omod/src/main/webapp/resources/scripts/taggedObjects.js

```diff
@@ -0,0 +1,42 @@
+
+var app = angular.module('taggedObjects',['personService','tagService']);
+
+app.config(['$locationProvider', function($locationProvider){
+    $locationProvider.html5Mode({
+        enabled: true,
+        requireBase: false
+    });
+}]);
+
+app.controller('taggedObjectsCtrl',['$scope','$location','Person','TagService', function ($scope,$location,Person,TagService) {
+
+    var tags = [];
+    $scope.init = function () {
+        $scope.tag = $location.search().tag;
+        loadTags();
+    }
+
+    var loadTags =function () {
+        var payload = {
+            objectType: 'org.openmrs.Person',
```

Comment:
I think this should be `org.openmrs.Patient` because not all persons are patients. You need to fix all the other areas where you've been using `org.openmrs.Person` in the previous commit